WooCommerce/Classes/View Modifiers/WooStyleModifiers.swift

```diff
@@ -0,0 +1,53 @@
+import Foundation
+import SwiftUI
+
+/// Woo style modifiers.
+/// Migrate them from `UILabel+Helpers` or  `UIButton+helpers` as needed.
+///
+
+// MARK: Body Style
+struct BodyStyle: ViewModifier {
+    func body(content: Content) -> some View {
+        content
+            .font(.body)
+            .foregroundColor(Color(.text))
+    }
+}
+
+extension View {
+    func bodyStyle() -> some View {
+        self.modifier(BodyStyle())
+    }
+}
```

Comment:
I was on the fence here, at the end I choose to have the modifier and the extension close to each other to be able to avoid unnecessary scrolling. but I'm fine either way!